docs/source/Advanced-Configuration.md

```diff
@@ -174,15 +174,41 @@ First, you need to configure the environment variable `LOCAL_STORAGE` to `False`
 Then you need to configure permission access to the chosen S3 bucket.
 
 
-#### SQS
+#### Message Broker
 
-If you like, you could use AWS SQS instead of Rabbit-MQ to manage your queues.
+IntelOwl at the moment supports 3 different message brokers:
+- RabbitMQ
+- Aws SQS
+- Redis
```

Comment:
```suggestion
- Redis (default)
- RabbitMQ
- Aws SQS
```

I know that it is written below but that's to be more more clear at the very start